packages/circuit-ui/components/Table/Table.tsx

```diff
@@ -60,6 +60,14 @@ export interface TableProps extends HTMLAttributes<HTMLDivElement> {
     currentRows: Row[],
     nextDirection?: Direction,
   ) => Row[];
+  /**
+   * Specifies the intial sort order of the table
+   */
+  initialSortDirection?: 'ascending' | 'descending';
+  /**
+   * Specifies the row index which initialSortDirection will be applied to
```

Comment:
By convention, we style prop names as code.

```suggestion
   * Specifies the row index which `initialSortDirection` will be applied to
```